packages/url/src/prepend-http.js

```diff
@@ -24,7 +24,7 @@ export function prependHTTP( url ) {
 
 	url = url.trim();
 	if ( ! USABLE_HREF_REGEXP.test( url ) && ! isEmail( url ) ) {
-		return 'http://' + url;
+		return 'https://' + url;
```

Comment:
We will also need to update some snapshots that previously used `http` and will now be using `https`.